test/controllers/integrations/github_controller_test.rb

```diff
@@ -8,7 +8,7 @@
 
   let(:payload) do
     {
-      ref: 'refs/heads/origin/dev',
```

Comment:
are you sure they are not coming in with the origin ? <-> did you check a real request before changing the fixture ?